wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/Reactions/ReactionToggle.swift

```diff
@@ -116,4 +122,15 @@ class ReactionToggle: UIControl {
         onToggle?()
     }
 
+    // MARK: - Accessibility
+
+    func setupAccessibility(
+        stackView: UIStackView,
+        value: String,
+        count: UInt
+    ) {
+        stackView.isAccessibilityElement = true
+        stackView.accessibilityIdentifier = "value: \(value), count: \(count)"
+    }
```

Comment:
Can we make the reaction toggle the accessibility element?

Reply:
Done!